### PR TITLE
Rendering table with data from service call

### PR DESCRIPTION
**Pull request by BraydonKains**

### CODE REVIEW

File: src/components/d2l-teacher-course-creation-admin.js [hunk 1]
```diff
@@ -36,23 +49,92 @@ class TeacherCourseCreationAdmin extends BaseMixin(LitElement) {
...
 
 	async connectedCallback() {
 		super.connectedCallback();
 
-		this.roles = await window.tccService.getRoles();
-		this.courseTypes = await window.tccService.getCourseTypes();
-		this.associations = await window.tccService.getAssociations();
+		this.roles = await tccService.getRoles();
+		this.courseTypes = await tccService.getCourseTypes();
+		this._fetchAssociations();
```
Comment (josephsaffron) [inline src/components/d2l-teacher-course-creation-admin.js:59]: I like it. =)

File: src/components/d2l-teacher-course-creation-admin.js [hunk 2]
```diff
@@ -36,23 +49,92 @@ class TeacherCourseCreationAdmin extends BaseMixin(LitElement) {
...
+		tccService
+			.getAssociations()
+			.then(associationsArray => {
+				let i = 0;
+				associationsArray.map(association => association.RowId = i++);
+				this.associations = associationsArray;
+			});
+	}
+
+	_handleAssociationEdit(event) {
```
Comment (josephsaffron) [inline src/components/d2l-teacher-course-creation-admin.js:72]: should we have one for AddNew too?

File: locales/en.js [hunk 3]
```diff
@@ -1,3 +1,9 @@
 export const val = {
+	'actionEdit': 'Edit',
```
Comment (josephsaffron) [inline locales/en.js:2]: not this PR, but we need to add this repo to the serge localize: https://git.dev.d2l/projects/RE/repos/serge-localize/browse

File: src/components/d2l-teacher-course-creation-admin.js (hunk 2 above)
Comment (BraydonKains) [reply src/components/d2l-teacher-course-creation-admin.js:72]: Yeah I'm going to add that Add New button in the next PR with the other dialogs.

**Pull request merged**